msvc/ut-build-early.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Starboard OneCore Overrides">
+    <_SB_Local_VCInstallDir Condition="'$(_SB_Local_VCInstallDir)' == ''">$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VC@ProductDir)</_SB_Local_VCInstallDir>
```

Comment:
They other reason I point this is out is, when VS 15 comes out, some poor sap, let's call him Artem, will be tasked with validating VS 15.  And, it will probably work because he'll have VS14 and VS15 side by side.  We'll sign off, and then somebody who only has VS15 installed will hit this.

But, if this is the only way, it's OK.  The VS targets/props are quite complex.  

---
In reply to: [74666079](https://github.com/Microsoft/WinObjC/pull/874#discussion_r74666079) [](ancestors = 74666079)